docs/data/toolpad/studio/reference/components/autocomplete.md

```diff
@@ -8,16 +8,16 @@ A text input with autocomplete suggestions. Uses the Material UI [Autocomplete]
+| Name                                        | Type                                   | Default                                     | Description                                                                                                                                                                                                                                                                          |
+| :------------------------------------------ | :------------------------------------- | :------------------------------------------ | :----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| <span class="prop-name">options</span>      | <span class="prop-type">array</span>   | <span class="prop-default">[]</span>        | The options available to search from.                                                                                                                                                                                                                                                |
+| <span class="prop-name">value</span>        | <span class="prop-type">string</span>  | <span class="prop-default">""</span>        | The value of the autocomplete.                                                                                                                                                                                                                                                       |
+| <span class="prop-name">defaultValue</span> | <span class="prop-type">string</span>  | <span class="prop-default">""</span>        | A default value.                                                                                                                                                                                                                                                                     |
+| <span class="prop-name">label</span>        | <span class="prop-type">string</span>  | <span class="prop-default">"Search…"</span> | The label to display for the autocomplete.                                                                                                                                                                                                                                           |
+| <span class="prop-name">fullWidth</span>    | <span class="prop-type">boolean</span> |                                             | If true, the autocomplete will take up the full width of its container.                                                                                                                                                                                                              |
```

Comment:
```suggestion
| <span class="prop-name">fullWidth</span>    | <span class="prop-type">boolean</span> |                                             | If true, the autocomplete takes up the full width of its container.                                                                                                                                                                                                              |
```

Reply:
Oh wait this isn't the right place to change it, ignore it, I'll add a commit.

Reply:
Yeah, I assumed this had to be done somehow automatically.